src/lib/injections.js

```diff
@@ -155,8 +203,8 @@ class Injections {
   }
 
   async disableContentScripts(injection) {
-    const contentScript = this._activeInjections.get(injection);
-    await contentScript.unregister();
+    const scriptId = this._activeInjections.get(injection);
```

Comment:
@Rob--W if we port (or when we port, if we are not going to do that in this PR) the shims to the scripting API then we would still to tell from a getRegisteredContentScripts result which are the ones registered from Injections and which are registered from Shims and so I think we would still need to keep _activeInjections 

If we want we could consider changing _activeInjections to be a Set of injection objects or injection ids instead of a Map if we want that.